pvlib/irradiance.py

```diff
@@ -1182,6 +1182,39 @@ def perez(surface_tilt, surface_azimuth, dhi, dni, dni_extra,
         return sky_diffuse
 
 
+def clearsky_index(ghi, clearsky_ghi, max_clearsky_index=2.0):
+    """
+    Calculate the clearsky index.
+
+    The clearsky index is the ratio of global to clearsky global irradiance.
+
+    Parameters
+    ----------
+    ghi : numeric
+        Global horizontal irradiance in W/m^2.
+
+    clearsky_ghi : numeric
+        Modeled clearsky GHI
+
+    max_clearsky_index : numeric, default 2.0
+        Maximum value of the clearsky index. The default, 2.0, allows
+        for over-irradiance events typically seen in sub-hourly data.
+
+    Returns
+    -------
+    kt : numeric
```

Comment:
This shouln't be called kt nor 'clearness index' as it creates potential confusion with the function below, which *does* calculate clearness index.

Reply:
@adriesse Good catch, thank you.  Careless copy/pasting.  I'm not aware of a shorthand symbol for clear-sky index, do you have any suggestions?

Reply:
@kevinsa5 I'd spell it out `clearsky_index : numeric` there are some common notations, e.g., Kt and k'_t but not a consensus AFAIK.